cadl-tests/cadl/builtin.cadl

```diff
@@ -35,6 +35,7 @@ model FormatString {
 
   @format("password")
   password: string;
+  uri: uri;
```

Comment:
please put `uri` in `Builtin`

`FormatString` is only for string, just different `@format`

Reply:
sure